### ISSUE TITLE
Use of undefined constant QM_EDITOR_COOKIE

### ISSUE THREAD

**Issue by siliconforks** [johnbillion/query-monitor#492]
On my development machine I'm seeing a lot of lines in the error log that look like this:

```
PHP Warning:  Use of undefined constant QM_EDITOR_COOKIE - assumed 'QM_EDITOR_COOKIE' (this will throw an Error in a future version of PHP)
```

This appears to be similar to issue #59 but the situation where it occurs is different: it happens when an error occurs in a different plugin and Query Monitor displays a stack trace.  Furthermore, that error must occur before Query Monitor's `plugins_loaded` hook runs.  I see three possible cases that can trigger this:

1. An error occurs in another plugin's `plugins_loaded` hook which was added with a priority less than 10 - for example, the following code will generate the `QM_EDITOR_COOKIE` warning:
    ```php
    add_action( 'plugins_loaded', function () {
        throw new \Exception( 'Testing exception handler' );
    }, 9 );
    ```
2. An error occurs in another plugin's `plugins_loaded` hook with priority 10, and that plugin has a name which comes before `query-monitor` alphabetically.
3. An error occurs in another plugin at the time the plugin is loaded (i.e., not in any hook), and that plugin has a name which comes after `query-monitor` alphabetically.

**Comment by johnbillion** [comment]
Good spot, thanks.

**Comment by flowdee** [comment]
Would be great if this could be fixed asap ;)

**Comment by johnbillion** [comment]
Fixed in 6047747 and 1417fdf.
